active/NavigationView/BottomItems/BottomMenuItems2.md

```diff
@@ -0,0 +1,194 @@
+[display mode](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes).
+See the left-aligned NavigationView example below for reference:
+
+![NavigationView with labels](./leftnav-anatomy2.png)
+
+NavigationView provides a built-in 
+[Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3), 
+which is always positioned at the bottom or right of the Pane, and a 
+[PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) 
+area that positions custom content adjacent to the Settings item.
+Although an app developer can place any content into PaneFooter, including other NavigationViewItems, 
+this content will not participate in the selection model and animations of 
+the MenuItems navigation list and Settings item. So if an item in MenuItems is selected, and the user
+clicks on an item in the PaneFooter, it will not de-select the preview item.
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
+
+Some apps may wish to visually group navigation items into top-aligned and bottom-aligned lists
+(or left- and right-aligned lists), for example:
+
+![NavigationView in Left display mode showing FooterMenuItems](FooterItems_Left.png)
+
+![NavigationView in Top display mode showing FooterMenuItems](FooterItems_Top.png)
+
+While this grouping is possible today by putting NavigationViewItems into the PaneFooter, 
+the approach creates a poor selection and interaction experience as described above.
+That's being corrected in this spec with a new FooterMenuItems property.
+
+As a result, keyboard and assistive technology users will feel that the footer list behavior meets
+their expectations -  the top and bottom menu items will be specified as two separate lists, 
+but it will be smooth to navigate from one to the other. See more in Accessibility section below.
+Selection behavior and animations will also feel better integrated into the whole NavigationView control, 
+as bottom menu items will participate in the same selection model as top menu items. 
+
+# Description
+
+## NavigationView.FooterMenuItems
+
+Use the FooterMenuItems to place navigation items at the end of the navigation pane,
+contrasted with the MenuItems property which places items at the start of the pane.
+
+```cs
+public IList<object> FooterMenuItems { get; }
+```
+
+The default Settings item will be included as the last item in the FooterMenuItems collection. 
+As a result, adding items into FooterMenuItems will by default display them before the Settings item. 
+The Settings item can still be toggled using the `IsSettingsVisible` property. 
+Note: this is the recommended way to remove the Settings item, although it's possible 
+to remove it manually by removing it from the FooterMenuItems collection itself. 
+
```

Comment:
Regarding the item is in both MenuItems and FooterItems point, wouldn't the VisualTree prevent this as an item can not be added to the visual tree if it is already is the child of another item? If an item somehow get's added to both collections, as soon as the NavigationView get's rendered, it would crash as this violates one of the rules of the visual tree. Of course one could define the same item twice, and add them to both, but that is nothing we would realistically be able to detect without major performance hits.

Reply:
Yeah, this feels OK to be no better or worse than trying to put an item into a ListView more than once.

Reply:
I believe the SettingsItem is not itself a UI element, but rather an Item, and you get the UI Element via ContainerFromMenuItem. So if you put SettingsItem in both MenuItems and FooterItems, it will get two containers, which can both go into the tree without error. (If you then call ContainerFromItem on the SettingsItem, you will get one or the other UI element but you can't predict which one comes out.)

Reply:
With the current implementation, the Settings item is created once and passed to the itemssource as UI element. So at all times, there exists exactly once. When we switch from Left navigation to top navigation, we are still using the same settings item. So when you try to add it to both, it will crash as the settingsitem is not created through a container.